setup.py

```diff
@@ -43,12 +43,10 @@ def read(*rnames):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Zope Public License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
```

Comment:
(BTW other people might be less than happy with the dropping of Python 2 at this time, see https://github.com/zopefoundation/meta/issues/15 and https://www.zope.org/developer/python2.html.)

Reply:
decided not to drop it.